src/reference/v8.json

```diff
@@ -97,6 +97,32 @@
         }
       }
     },
+    "state": {
+      "type": "state",
+      "default": {},
+      "doc": "An object defining global state properties of the map style. State property value can be retrieved with the [`global-state`](https://maplibre.org/maplibre-style-spec/expressions/#global-state) expression.\n\n State is required when `global-state` is used in the style.",
+      "example": {
+        "chargerType": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "default": ["CCS", "CHAdeMO", "Type2"]
+        },
+        "minPreferredChargingSpeed": {
+          "type": "number",
+          "minimum": 0,
+          "default": 50
+        }
+      },
+      "sdk-support": {
+        "basic functionality": {
+          "js": "https://github.com/maplibre/maplibre-gl-js/issues/4964",
+          "android": "https://github.com/maplibre/maplibre-native/issues/3302",
+          "ios": "https://github.com/maplibre/maplibre-native/issues/3302"
```

Comment:
Please include some additional information in the Native ticket. Can just be copy pasted, but issues should be self-contained.